modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/_masturbation_item.dm

```diff
@@ -0,0 +1,68 @@
+/obj/item/hand_item/coom
+	name = "cum"
+	desc = "C-can I watch...?"
+	icon = 'icons/obj/hydroponics/harvest.dmi'
+	icon_state = "eggplant"
+	inhand_icon_state = "nothing"
+
+/obj/item/hand_item/coom/attack(mob/living/target, mob/user, proximity)
+	. = ..()
+	do_masturbate(target, user, proximity)
+
+// Jerk off into bottles
+/obj/item/hand_item/coom/afterattack(obj/target, mob/user, proximity)
+	. = ..()
+	do_masturbate(target, user, proximity)
```

Comment:
Why does it do it on attack _and_ afterattack?

Reply:
Good question, I turned off my brain for this code when I was copy-pasting.